developers/weaviate/manage-data/classes.mdx

```diff
@@ -85,25 +85,28 @@ To create a class, you must specify the class name with the `class` parameter.
   </TabItem>
 </Tabs>
 
-Typically, you would specify additional configurations. We include some examples in the following sections.
+Typically, you would specify additional configurations. We include examples in the following sections.
+
+
+#### Define Properties
+
+The class definition can include properties, which are the data fields to be stored in Weaviate. Each property must have a name and a data type. The `Auto-schema` feature can create properties at import time and infer the data type from the data, but we recommend you specify the data type manually.
 
 ### Sample configuration
 
-One example configuration is shown below, which defines:
+A class configuration is shown below, which defines:
 - The class name (`Article`)
 - The [vectorizer module](../modules/retriever-vectorizer-modules/index.md) (`text2vec-cohere`)
-- The [vector distance metric](../config-refs/distances.md) (`cosine`)
 - Which [generative module](../modules/reader-generator-modules/index.md) to use (`generative-openai`)
-- A set of [properties](../concepts/data.md#data-object-nomenclature) (`title`, `chunk`, `chunk_no` and `url`), with [data types](../config-refs/datatypes.md) set accordingly.
+- A set of [properties](../concepts/data.md#data-object-nomenclature) (`title`, `body` and `url`), with [data types](../config-refs/datatypes.md) set accordingly.
+    - Note that the `body` property is an array of objects.
```

Comment:
This note should be part of the properties entry rather than a stand alone entry